starter-code/pizza.js

```diff
@@ -1 +1,209 @@
 // Write your Pizza Builder JavaScript in this file.
+
+
+
+//when document is loaded, user has to see suce and gluten-free button disabled. Pizza foto has to be without white sauce and gluten-free pizza base.
+
+$(document).ready(function () {
+
+  //whit this code i do not have to edit HTML file and manualy delete class that HTML tag has.
+  //after this two if statements I am sure that buttons want be active.
+  if ($(".btn-sauce").hasClass("active")) {
+    $(".btn-sauce").removeClass("active");
+  }
+
+  if ($(".btn-crust").hasClass("active")) {
+    $(".btn-crust").removeClass("active");
+  }
+
+  //pizza foto has to be without white sauce and gluten-free crust.
+
+  $(".sauce").removeClass("sauce-white");
+  $(".crust").removeClass("crust-gluten-free");
+  $("strong").text("$13"); // basoc price.
+  $("ul:last > li:nth-child(4)").hide(); // price for ingredients which are not included has to be removed from list on right side.
+  $("ul:last > li:nth-child(5)").hide(); // price for ingredients which are not included has to be removed from list on right side.
+});
+
```

Comment:
I think whats missing here are functions and variables that show whats going on.
For example. the top of you file I would declare all your buttons eg

 ` var pepperoniButton = $(".btn-pepperonni")` etc..
``` 
function setDefaultPizza() {
   ... remove sauce 
   ... remove GF crusts
}

function removeTopping( parameter ) {
   ... here is the challenge of being able to use this function for all toppings.  You can use class names to select 
   items on the pizza based on the button you click.
}

function addTopping( parameter ) {
   ... etc
}
```